universal-application-tool-0.0.1/test/repository/WithResettingPostgresContainer.java

```diff
@@ -0,0 +1,19 @@
+package repository;
+
+import io.ebean.Ebean;
+import io.ebean.EbeanServer;
+import models.Applicant;
+import models.Person;
+import models.Program;
+import models.Question;
+import org.junit.Before;
+import play.db.ebean.EbeanConfig;
+
+public class WithResettingPostgresContainer extends WithPostgresContainer {
```

Comment:
question: Is there any reason we now need both of these? Or can we reset the other WithPostgresContainer and just have the one?

Reply:
the `FunctionalTest` requires state. I think that's not good, but that's the way it is. We can address that later.